ruby/example_code/cloudtrail/README.md

```diff
@@ -1,61 +1,56 @@
-# AWS SDK for Ruby code examples for AWS CloudTrail
+# CloudTrail code examples for the AWS SDK for Ruby
+## Overview
+These examples show how to create and manage AWS CloudTrail trails using the AWS SDK for Ruby.
 
-## Purpose
-This folder contains code examples that demonstrate how to use the AWS SDK for Ruby to automate AWS CloudTrail.
+With AWS CloudTrail, you can monitor your AWS deployments in the cloud by getting a history of AWS API calls for your account.
 
-CloudTrail is an AWS service that helps you enable governance, compliance, and operational and risk auditing 
-of your AWS account. Actions taken by a user, role, or an AWS service are recorded as events in CloudTrail.
+## ⚠️ Important
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
 
 ## Code examples
 
-### API examples
-- [Create a trail](./create_trail.rb)
-- [Delete a trail](./delete_trail.rb)
-- [Describe trails](./describe_trails.rb)
-- [Lookup events](./lookup_events.rb)
+### Single actions
+Code excerpts that show you how to call individual service functions.
 
-## Important
+* [Create a trail](./create_trail.rb) (`CreateTrail`)
 
-- As an AWS best practice, grant this code least privilege, or only the 
-  permissions required to perform a task. For more information, see 
-  [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) 
-  in the *AWS Identity and Access Management User Guide*.
-- This code has not been tested in all AWS Regions. Some AWS services are 
-  available only in specific AWS Regions. For more information, see the 
-  [AWS Regional Services List](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/)
-  on the AWS website.
-- Running this code might result in charges to your AWS account.
+* [Delete a trail](./delete_trail.rb) (`DeleteTrail`)
 
-## Running the code
+* [Describe trails](./describe_trails.rb) (`DescribeTrails`)
+
+* [Lookup events](./lookup_events.rb) (`LookupEvents)
+
+
+
+
+
+
```

Comment:
Throughout the README files, I'm seeing these extra lines between the Code examples and Run the examples sections.

Reply:
This does not render in the README and is happening for reasons I can't explain or resolve.